pkg/operator/ceph/config/conditions.go

```diff
@@ -117,28 +117,32 @@ func translatePhasetoState(phase cephv1.ConditionType) cephv1.ClusterState {
 	}
 }
 
-// Updating the status of Progressing, Updating or Upgrading to False once cluster is Ready
-func checkConditionFalse(context *clusterd.Context, namespace, name string) {
+// MakeConditionFalse Updates the status of Progressing, Updating or Upgrading to False once cluster is Ready
+func MakeConditionFalse(context *clusterd.Context, namespace, name string) {
```

Comment:
What about naming the method `MarkProgressConditionsCompleted()`? Seems like that's what is happening. Then you don't have to think about what `false` means.

Reply:
Done.